test/e2e/test-data/eraser_v1alpha1_imagelist.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: eraser.sh/v1alpha1
+kind: ImageList
+metadata:
+  name: imagelist-sample
+  namespace: eraser-system
```

Comment:
ImageList and ImageJob shouldn't be namespaced https://github.com/Azure/eraser/blob/main/config/crd/bases/eraser.sh_imagelists.yaml#L17